src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothHidDevice.cs

```diff
@@ -0,0 +1,248 @@
+    /// The Host(BluetoothHid) is a device that uses or requests the service of a HID.
+    /// The Device(BluetoothHidDevice) is a device that provides the service of human data input/output to/from the host.
+    /// </remarks>
+    /// <privilege> http://tizen.org/privilege/bluetooth </privilege>
+    /// <privilege> http://tizen.org/privilege/bluetooth.hid_device </privilege>
+    /// <since_tizen> 6 </since_tizen>
+    public class BluetoothHidDevice : BluetoothProfile
+    {
+        private event EventHandler<HidDeviceConnectionStateChangedEventArgs> _hidDeviceConnectionStateChanged;
+        private event EventHandler<HidDeviceDataReceivedEventArgs> _hidDeviceDataReceived;
+
+        internal BluetoothHidDevice()
+        {
+        }
+
+        /// <summary>
+        /// The ConnectionStateChanged event is called when the HID device connection state is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<HidDeviceConnectionStateChangedEventArgs> ConnectionStateChanged
+        {
+            add
+            {
+                _hidDeviceConnectionStateChanged += value;
+            }
+            remove
+            {
+                _hidDeviceConnectionStateChanged -= value;
+            }
+        }
+
+        /// <summary>
+        /// Activates the Bluetooth HID Device role.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.
+        /// or when the connection attempt to the remote device fails.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void Activate()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                Interop.Bluetooth.HidDeviceConnectionStateChangedCallback _hidDeviceConnectionStateChangedCallback = (int result, bool isConnected, string address, IntPtr userData) =>
+                {
+                    _hidDeviceConnectionStateChanged?.Invoke(null, new HidDeviceConnectionStateChangedEventArgs(result, isConnected, address));
+                };
+
+                int ret = Interop.Bluetooth.ActivateHidDevice(_hidDeviceConnectionStateChangedCallback, IntPtr.Zero);
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to activate to the remote device, Error - " + (BluetoothError)ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Deactivates the Bluetooth HID Device role.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.
+        /// or when the connection attempt to the remote device fails.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void Deactivate()
```

Comment:
It should be static, because it influences all HidDevice objects

Reply:
Done.